### PR TITLE
PMM-7145 New default

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**

### CODE REVIEW

File: docs/details/commands/pmm-admin.md [hunk 1]
```diff
@@ -139,8 +139,8 @@ PMM communicates with the PMM Server via a PMM agent process.
 
     `--metrics-mode=mode`
     : Metrics flow mode for agents node-exporter. Allowed values:
-        - `auto`: chosen by server (default)
-        - `push`: agent will push metrics
+        - `auto`: chosen by server
```
Comment (BupycHuk) [inline docs/details/commands/pmm-admin.md:142]: `auto` is still default one.

Comment (BupycHuk) [reply docs/details/commands/pmm-admin.md:142]: on the server side we chose push, but on pmm-admin side `auto` is still the default value.

Comment (PaulJacobs-percona) [reply docs/details/commands/pmm-admin.md:142]: Then nothing changes in pmm-admin man page? (Every is 'auto' default).

Comment (BupycHuk) [reply docs/details/commands/pmm-admin.md:142]: yes, we can close this PR.

**Pull request closed without merging**